src/solver/conv_hip_implicit_gemm_3d_grouped_bwd_xdlops.cpp

```diff
@@ -123,6 +124,29 @@ struct CKArgs
     template <typename ConvPtr>
     auto MakeArgPtr(const ConvPtr& conv_ptr, ConstData_t in, ConstData_t w, Data_t out) const
     {
+
+#if 0 // Leaving for debugging needs
+        std::cout << "y ptr = " << in << std::endl;
+        std::cout << "w ptr = " << w << std::endl;
+        std::cout << "x ptr = " << out << std::endl;
+
+        auto print_vec = [](const char* name, const auto& vec) {
+            std::cout << name << " = [ ";
+            for(const auto& v : vec)
+            {
+                std::cout << v << ", ";
+            }
+            std::cout << "]\n";
+        };
+#define PRINT_VEC(x) print_vec(#x, x);
+
+        PRINT_VEC(output);
+        PRINT_VEC(out_strides);
+        PRINT_VEC(input);
+        PRINT_VEC(in_strides);
+        PRINT_VEC(weight);
+        PRINT_VEC(wei_strides);
+#endif
```

Comment:
So far we have never had such code in MIOpen, if we intend on keeping this, we should move the macro to the proper header file. and add it as a higher log level.